lfa_lab/core/NdRange.i

```diff
@@ -41,7 +41,7 @@ class stop_iteration : public std::exception { };
                 m_pos(grid.begin())
             { }
 
-            ArrayFi next() {
+            ArrayFi __next__() {
```

Comment:
Does this still work in Python 2?

Reply:
Unfortunately not. Though, there is a way to make an iterator work both in Python 2 and 3. One can just stick for the Python two syntax (next() without underscore) and then add the following assignment to the class:
```python
__next__ = next
```
Although I'm not that familiar with SWIG and hence have no idea how to automatically generate this additional assignment.